.mvn/extensions.xml

```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+  <extension>
+    <groupId>io.jenkins.tools</groupId>
+    <artifactId>git-changelist-maven-extension</artifactId>
```

Comment:
Would you let me know how this extension work? There look no documentations about this extension.
I wonder the version rewriting in `release:prepare` and `release:perform` works correct.

Reply:
Thanks! I got what incremental is.

So `useBeta=true` only for git-changelist-maven-extension and not for workflow-api.
I will remove `useBeta=true` when the actual version of  git-changelist-maven-extension is released.